widgetsnbextension/src/extension.js

```diff
@@ -46,7 +50,7 @@ var handle_cell = function(cell) {
     }
 };
 
-function register_events(Jupyter, events) {
+function register_events(Jupyter, events, outputarea) {
```

Comment:
@jasongrout Yep, this is cleaner. Before we unwebpackified the notebook, doing this didn't work. I will update the mimerender extensions to reflect this :+1: